framework/include/cppmicroservices/detail/ServiceTracker.tpp

```diff
@@ -143,36 +143,26 @@ template<class S, class T>
 void ServiceTracker<S,T>::Close()
 {
   std::vector<ServiceReference<S>> references;
-  std::shared_ptr<_TrackedService> outgoing = d->trackedService.Exchange(std::shared_ptr<_TrackedService>());
-  if (outgoing == nullptr)
-  {
+  std::shared_ptr<_TrackedService> outgoing = d->trackedService.Load();
+  if (outgoing == nullptr) {
     return;
   }
 
   DIAG_LOG(*d->context.GetLogSink()) << "ServiceTracker<S,TTT>::close:" << d->filter;
   outgoing->Close();
-  references = GetServiceReferences();
-  try
-  {
+
+  try {
     d->context.RemoveListener(std::move(d->listenerToken));
-  }
-  catch (const std::runtime_error& /*e*/)
-  {
+  } catch (const std::runtime_error& /*e*/) {
     /* In case the context was stopped or invalid. */
   }
 
   d->Modified(); /* clear the cache */
   outgoing->NotifyAll(); /* wake up any waiters */
-  for(auto& ref : references)
-  {
-    outgoing->Untrack(ref, ServiceEvent());
-  }
```

Comment:
Does this mean that `ServiceTrackerCustomizer`s will no longer get a call to `ServiceRemoved` for every tracked service when the `ServiceTracker` is closed?